src/Lykke.AlgoStore.Api/Infrastructure/Extensions/ErrorExtensions.cs

```diff
@@ -21,7 +21,9 @@ public static ObjectResult ToHttpStatusCode(this AlgoStoreException error)
             errorResponse.ErrorCode = (int)error.ErrorCode;
             errorResponse.ErrorDescription = error.ErrorCode.ToString("g");
             errorResponse.ErrorMessage = error.Message;
-            errorResponse.DisplayMessage = error.DisplayMessage;
+            errorResponse.DisplayMessage = string.IsNullOrEmpty(error.DisplayMessage)
+                ? (string.IsNullOrEmpty(error.ErrorMessage) ? error.Message : error.ErrorMessage)
```

Comment:
should we show the Error message to the user... if it is an exception it will be a problem...we should show only valid and logical errors to the user. We also don't have to show ids or such info...maybe something like -  "We have a technical problem, please contact the administrator" is a better idea, than displaying the exception message.